dna--1.0.sql

```diff
@@ -115,10 +115,18 @@ CREATE OPERATOR <-> (
   AS 'MODULE_PATHNAME', 'length'
   LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
 
+/******************************************************************************
+* K-mers
+******************************************************************************/
 
-  /******************************************************************************
-  * For Qkmer 
- ******************************************************************************/
+CREATE FUNCTION generate_kmers(dna dna, k int)

```

Comment:
First arg is cast from string to DNA with "dna_cast_from_text" directly - gotta love pg magic